### PR TITLE
SPLAT-1592: bump capv to v1.9.3

### PR DESCRIPTION
**Pull request by jcpowermac**

### CODE REVIEW

**Comment by patrickdillon** [comment]
you're going to want to want to update the infrastructure components file: https://github.com/openshift/installer/tree/master/data/data/cluster-api (unfortunately you will need to manually copy it) and you should also probably bump the installer's capv 
https://github.com/openshift/installer/blob/5578855cd93d99fb07ba5bf70d598e298b072bdf/go.mod#L121

File: vendor/modules.txt [hunk 1]
```diff
@@ -1390,7 +1390,7 @@ golang.org/x/mod/internal/lazyregexp
 golang.org/x/mod/modfile
 golang.org/x/mod/module
 golang.org/x/mod/semver
-# golang.org/x/net v0.22.0 => golang.org/x/net v0.5.0
+# golang.org/x/net v0.23.0 => golang.org/x/net v0.5.0
```
Comment (r4f4) [inline vendor/modules.txt]: I'm surprised this is not causing problems. I'll try to fix in a follow-up.

Comment (r4f4) [reply vendor/modules.txt]: https://github.com/openshift/installer/pull/8271

**Pull request merged**